src/peergos/shared/user/TofuKeyStore.java

```diff
@@ -57,6 +64,8 @@ public TofuKeyStore() {
                     // we are allowing expired chains to be stored
 
                     List<UserPublicKeyLink> existing = getChain(username);
+                    if (! existing.isEmpty() && existing.get(existing.size() - 1).equals(tail.get(tail.size() - 1)))
```

Comment:
not sure i understand this

Reply:
This is just saying if nothing has changed for that user in the pki then we don't need to commit the tofukeystore.